core/src/main/java/fr/pilato/elasticsearch/crawler/fs/FsParserAbstract.java

```diff
@@ -97,12 +99,10 @@
             messageDigest = null;
         }
 
-        // On Windows, when using SSH server, we need to force the "Linux" separator
-        if (OsValidator.WINDOWS && fsSettings.getServer() != null) {
-            logger.debug("We are running on Windows with SSH Server settings so we need to force the Linux separator.");
-            pathSeparator = "/";
-        } else {
-            pathSeparator = File.separator;
+        pathSeparator = FsCrawlerUtil.getPathSeparator(fsSettings.getFs().getUrl());
```

Comment:
This is a good idea IMO but this will be a breaking change I think. Some people on windows might use `c:/tmp` as documented here: https://fscrawler.readthedocs.io/en/latest/admin/fs/local-fs.html#id2

Wondering if we should change the documentation?

Reply:
I've tested `c:/tmp` and `c:\\tmp` on my Windows local crawler(actually I've tested all crawlers) and it's [compatible](https://github.com/dadoonet/fscrawler/pull/1203/files#diff-27aabc48cd80adbdd85f89c53ff70563163da6ed75cb3b72251179dad706c038R285). Path is also [resolved](https://github.com/dadoonet/fscrawler/pull/1203/files#diff-7244908aba4d45bf4dc283a08fa10d30700fb5f6a7633f8c8b35937c1d122d48R54) and saved in accordance with user input in this scenario.

Reply:
Great!